database/src/test/scala/za/co/absa/atum/database/runs/GetPartitioningMeasuresTest.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.database.runs
+
+import za.co.absa.balta.DBTestSuite
+import za.co.absa.balta.classes.JsonBString
+
+
+class GetPartitioningMeasuresTest extends DBTestSuite{
+
+  private val fncGetPartitioningMeasures = "runs.get_partitioning_measures"
+
+  private val partitioning = JsonBString(
+    """
+      |{
+      |   "version": 1,
+      |   "keys": ["key1", "key2"],
+      |   "values": {
+      |     "key1": "value1",
+      |     "key2": "value2"
+      |   }
+      |}
+      |""".stripMargin
+  )
+
+  test("Get partitioning measures") {
+    function(fncGetPartitioningMeasures)
+      .setParam("i_partitioning", partitioning)
+      .execute { queryResult =>
+        val results = queryResult.distinct.next()
+        assert(results.getInt("status").contains(11))
+        assert(results.getString("status_text").contains("OK"))
```

Comment:
I would also check which measures were returned. But for this you have to import them into the DB. Also you have to import partitioning into the DB, which is not done yet.

Balta has a very nice interface for this, please see WriteCheckpointTest file, and function calls like `table` . `import` or something like that